CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## 0.13.0 (2023-11-10)
+
+This release:
+- Bumps `scale-decode` to its latest version.
```

Comment:
I don't get why we follow http://keepachangelog.com/en/1.0.0/ format

For this I would expect it be

```bash
## Changed
 - Bump `scale-decode` to its latest version.
 ```

```

Reply:
Yeah; I've just been lazy about these changes so far; probably we should do it "more properly" at some point :)